lib/goal/cache/NoOpGoalCache.ts

```diff
@@ -26,9 +27,11 @@ import { GoalCache } from "./goalCaching";
  */
 export class NoOpGoalCache implements GoalCache {
     public async put(gi: GoalInvocation, project: GitProject, files: string[], classifier?: string): Promise<void> {
+        logger.debug(`Warning: No-Op goal cache in use; no cache will be preserved!`);
```

Comment:
Log on warn level and remove the word warning from the message. All debug message won’t show on the console much longer by default.

Reply:
Updated.